src/collections/_documentation/clients/cocoa/advanced.md

```diff
@@ -147,6 +147,16 @@ Client.shared?.snapshotStacktrace {
 }
 ```
 
+## Determining whether to send events
+
+If you want to filter certain events, you can use `shouldSendEvent` which takes the event, returns a boolean, and decides whether or not to send the event.
+
+```swift
+    self.shouldSendEvent = ^BOOL(SentryEvent *_Nonnull event) {
+        return (sampleRate >= ((double)arc4random() / 0x100000000));
+    };
+```
```

Comment:
So this code is objective c and since most code samples on this page are swift we should also have a swift example here.

```swift
        Client.shared?.shouldSendEvent = { (event) in
            if (event.environment == "staging") {
                return false
            }
            return true
        }
```